app/controllers/team_controller.rb

```diff
@@ -0,0 +1,13 @@
+class TeamController < ApplicationController
+
+    before_action :authenticate_user!
+
+    def index
+       @users= User.all
```

Comment:
I dont see the difference between team controller and users controller.
It would make sense if you had some team id for users. But in this case, users controller is good enough